_docs/guides/bookinfo.md

```diff
@@ -45,34 +39,63 @@ The end-to-end architecture of the application is shown below.
 <figcaption>BookInfo Application without Istio</figcaption></figure>
 
 This application is polyglot, i.e., the microservices are written in different languages.
+It’s worth noting that these services have no dependencies on Istio, but make an interesting
+sevice mesh example, particularly because of the multitude of services, languages and versions
+for the reviews service.
+
+## Deploying the application
+
+To run the sample with Istio requires no changes to the
+application itself. Instead, we simply need to configure and run the services in an
+Istio-enabled environment, with Envoy sidecars injected along side each service.
+The needed commands and configuration vary depending on the runtime environment
+although in all cases the resulting deployment will look like this:
+
+<figure><img src="./img/bookinfo/withistio.svg" alt="BookInfo Application" title="BookInfo Application" />
+<figcaption>BookInfo Application</figcaption></figure>
 
-## Start the application
+All of the microservices will be packaged with an Envoy sidecar that intercepts incoming
+and outgoing calls for the services, providing the hooks needed to externally control,
+via the Istio control plane, routing, telemetry collection, and policy enforcement
+for the application as a whole.
+
+Before you begin, if you haven't already done so, setup Istio by following the instructions
+corresponding to your platform [installation guide]({{home}}/docs/setup/) and then
```

Comment:
link directly https://istio.io/docs/setup/kubernetes/quick-start.html
(this doc is already k8s scoped, no need to have users need 2 more clicks to get to it)